exercises/am_i_ready/exercise.js

```diff
@@ -160,11 +160,12 @@ function checkGcc (pass, callback) {
 
 function checkMsvc (pass, callback) {
   var msvsVars    = {
-          2013: 'VS130COMNTOOLS'
+        2015: 'VSSDK140Install'
```

Comment:
Could use VS140COMNTOOLS instead.  With this change, you probably wouldn't need to change line 180/181.